scripts/benchmark-rule.mjs

```diff
@@ -24,17 +22,22 @@ function printHelp() {
 	console.log('  # With secondary options (JSON format)');
 	console.log(`  ${script} value-keyword-case '["lower", {"camelCaseSvgKeywords": true}]'`);
 	console.log('');
-	console.log('  # With a context');
+	console.log('  # With a config');
 	console.log(
 		`  ${script} value-keyword-case '["lower", {"camelCaseSvgKeywords": true}]' '{"fix": true}'`,
 	);
 }
 
-if (!ruleName || !(ruleName in rules) || !ruleOptions) {
+if (!ruleName || !ruleOptions) {
 	printHelp();
 	exit(-1);
 }
 
+if (!stylelint.rules[ruleName]) {
+	console.error(`Unknown rule: ${ruleName}`);
+	exit(-1);
```

Comment:
Should this be `exit(1)`?
Maybe the negative non-zero exit code is intentional :)

Reply:
Ah, you're entirely right. I just copied it from L33 without any thoughts. I'll fix it. 😅

Reply:
Fixed via 5a9fc2868.